app/api/api_v1/routers/geographies.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from fastapi import APIRouter, Depends
+
+from app.api.api_v1.schemas.geography import Geography
+from app.db.session import get_db
+
+_LOGGER = logging.getLogger(__file__)
+
+geographies_router = APIRouter()
+
+
+@geographies_router.get("/geographies", response_model=list[Geography])
+async def geographies(db=Depends(get_db)):
+    """Get a summary of all geographies for world map."""
+    _LOGGER.info("Getting detailed information on all geographies")
+
+    return [
```

Comment:
This looks good as a skeleton to allow the frontend to start using it, but do you have details on the next steps? Thinking that if we deploy this to production we can probably bet that no one will find the api link, but probably wouldn't want it there as a skeleton for too long all the same

Reply:
Fred and I have spoken in Slack about this in more detail and he is happy with the current approach (as the next PR for this ticket will be raised immediately after this has merged). He's happy for me to resolve these conversations in the meantime